runtime-modules/proposals/discussion/src/lib.rs

```diff
@@ -228,7 +243,13 @@ decl_module! {
        }
 
         /// Changes thread permission mode.
```

Comment:
Consider use Parity style T::WeightInfo and if it is impossible - introduce an type alias.